go.mod

```diff
@@ -78,3 +78,5 @@ require (
 	google.golang.org/protobuf v1.33.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace goauthentik.io/api/v3 => ./gen-go-api
```

Comment:
`git add .` gone wrong

Reply:
Ah shit yeah, need to figure out a better way to do this without having to remember to not add this file